ckanext/ontario_theme/templates/internal/package/snippets/resource_edit_form.html

```diff
@@ -0,0 +1,14 @@
+{% set stage = stage or 1 %}
+{% set include_metadata = include_metadata or false %}
+
+{% extends resource_form_snippet %}
+
+{% block stages %}{% endblock %}
+
+{% block previous_button %}{% endblock %}
+
+{% block again_button %}{% endblock %}
+
```

Comment:
You don't need this entire bit of code. Using `{% ckan_extends %}` here and using `{% block save_button %}` works.